xcode/Subconscious/Shared/Services/DatabaseService.swift

```diff
@@ -844,7 +853,7 @@ final class DatabaseService {
 extension Config {
     static let migrations = Migrations([
         SQLMigration(
-            version: Int.from(iso8601String: "2023-02-15T10:23:00")!,
+            version: Int.from(iso8601String: "2023-02-28T13:21:00")!,
```

Comment:
Bump database schema version. We've changed the way `MemoAddress` is serialized via `description`, which means the type has effectively changed for the database as well.

Bumping the schema will cause the database to be rebuilt, and notes to be reindexed.